opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -161,7 +161,15 @@ namespace Opm
             // Set it up manually
             ElementMapper elemMapper(simulator_.vanguard().gridView(), Dune::mcmgElementLayout());
             detail::findOverlapAndInterior(simulator_.vanguard().grid(), elemMapper, overlapRows_, interiorRows_);
-
+            numJacobiBlocks_ = EWOMS_GET_PARAM(TypeTag, int, NumJacobiBlocks);
+            useWellConn_ = EWOMS_GET_PARAM(TypeTag, bool, MatrixAddWellContributions);
+            if (numJacobiBlocks_ > 1) {
```

Comment:
Condition is not consistent with the one in the vanguard

Reply:
0ea1f86